src/NexusMods.Abstractions.Loadouts/Models/LoadoutObservables.cs

```diff
@@ -1,35 +1,71 @@
-using System.Collections.Frozen;
 using System.Reactive.Linq;
-using DynamicData;
-using NexusMods.Abstractions.MnemonicDB.Analyzers;
+using NexusMods.Abstractions.Loadouts.Rows;
+using NexusMods.Cascade;
+using NexusMods.Cascade.Flows;
+using NexusMods.Cascade.Patterns;
 using NexusMods.MnemonicDB.Abstractions;
+using NexusMods.MnemonicDB.Abstractions.Cascade;
 
 namespace NexusMods.Abstractions.Loadouts;
 
+
 public partial class Loadout
 {
     /// <summary>
-    /// Returns all the loadouts in the database, refreshing them whenever a child entity is updated.
+    ///  Returns all items in a loadout
     /// </summary>
-    public static IObservable<IChangeSet<Loadout.ReadOnly>> ObserveAllWithChildUpdates(IConnection connection)
-    {
-        var analyzerDatas = connection.Revisions.Select(db => db.AnalyzerData<TreeAnalyzer, FrozenSet<EntityId>>());
-        
-        return Loadout.ObserveAll(connection)
-            .AutoRefreshOnObservable(itm => analyzerDatas.Where(set => set.Contains(itm.Id)))
-            .RemoveKey();
-    }
+    private static readonly Flow<(EntityId Loadout, EntityId Entity)> LoadoutItemsSubFlow =
+        Pattern.Create()
+            .Db(out var loadoutItem, LoadoutItem.LoadoutId, out var loadoutId)
+            .Return(loadoutId, loadoutItem);
+
+    /// <summary>
+    /// Include all sort order entities that are associated with a loadout
+    /// </summary>
+    private static readonly Flow<(EntityId Loadout, EntityId Entity)> LoadoutSortOrderSubFlow =
+        Pattern.Create()
+            .Db(out var sortItem, SortOrder.LoadoutId, out var loadoutId)
+            .Return(loadoutId, sortItem);
```

Comment:
This returns `SortOrder` entities, which are just the parents of the actual `SortableEntries`, which are the entities that have the `SortIndex` and thus would change as the load order is updated.

I think this query is supposed to return the `SortableEntries` that have a `SortableEntry.ParentSortOrder` that references `loadoutId`.

Reply:
That makes sense, updated it to use the items themselves.